### ISSUE TITLE
`FsAutocomplete` no longer supports the `--enable-background-services` flag

### ISSUE THREAD

**Issue by OmanF** [ionide/Ionide-vim#61]
**Describe the bug**
The instructions call for setting the plugin with the flag `--enable-background-services` (both locally, and globally, installed `FsAutocomplete`), however when running `fsautocomplete` from the CLI, it errors, showing the flag is no longer supported.

As a result, the plugin doesn't trigger LSP integration with NVIM.

**To Reproduce**
1. Follow installing the plugin (`Ionide-vim`) according to the README file.
2. Make sure to set the correct behavior for the `fsharp#fsautocomplete_command` section of the settings, according to whether you have `fsautocomplete` installed locally or globally.

**Expected behavior**
No errors in the plugin integration with both `fsautocomplete` and Nvim.

**Environment (please complete the following information):**
 - OS: ArchLinux (latest, updated every day at noon).
 - Vim / Neovim version: 0.8.0 (`Release` build type).
 - dotnet SDK version: 6.0.110.
 - mono / .Net Framework version: N/A

**Additional context**
For some reason, when opening `.fs` or `.fsx`, or `.fsi` files, Nvim's built-in LSP client recognizes them as Forth files, not F# ones.

**Comment by baronfel** [comment]
That flag was removed in recent versions of FSAC - it was used to spawn a background check process that was removed from the service. It should be removed from the command spawn args if possible.

**Comment by OmanF** [comment]
I'll describe what I've tried and my Nvim config:
* I install the plugin with `Plug 'ionide/Ionide-vim'` (I took off the `make fsautocomplete` according to the README).
* On a project, I installed `fsautocomplete` locally (`dotnet tool install fsautocomplete`) and in my config file I have:
```
let g:fsharp#fsautocomplete_command =
  \ [ 'dotnet',
  \   'fsautocomplete'
  \ ]
```
* I then *uninstalled* `fsautocomplete` locally, and installed it globally (`dotnet tool uninstall fsautocomplete && dotnet tool install -g fsautcomplete`), and in my config file I put: `let g:fsharp#fsautocomplete_command = [ 'fsautocomplete' ]`

This covers both behaviors: local, and globally installed `fsautocomplete`.
As I said, neither integrates well, and the LSP client says the `.fs` file is a Forth file.

(Let me know what more needs testing. :D)
